_posts/2023-01-23-gsoc-2023.md

```diff
@@ -0,0 +1,12 @@
+---
+category: blog-detail
+post-type: blog
+by: Scala Center
+title: "Calls for Scala GSoC 2023 projects!"
+---
+
+Calls for open source Scala projects for [Google Summer of Code 2023](https://summerofcode.withgoogle.com/)!
```

Comment:
I guess you're assuming it's common knowledge what GSoC even is — or that people will follow the link if they don't know. Regardless, I'd suggest including some brief wording like: "GSoC is an internship program where Google funds students to work on open-source projects."

Reply:
s/Calls for/Call for/

Reply:
Done. Added short intro "GSoC is an online mentorship program that matches beginner contributors to open source software development with mentors." according to https://youtu.be/7jD2tChhrWM?t=38